project/Dependencies.scala

```diff
@@ -79,14 +91,16 @@ object Dependencies {
     val reactiveStreams = "org.reactivestreams" % "reactive-streams" % reactiveStreamsVersion // CC0
 
     // ssl-config
-    val sslConfigCore = "com.typesafe" %% "ssl-config-core" % sslConfigVersion // ApacheV2
+    // TODO DOTTY
+    val sslConfigCore = DottyCompatModuleID("com.typesafe" %% "ssl-config-core" % sslConfigVersion).withDottyCompat(getVersion()) // ApacheV2
 
     val lmdb = "org.lmdbjava" % "lmdbjava" % "0.7.0" // ApacheV2, OpenLDAP Public License
 
     val junit = "junit" % "junit" % junitVersion // Common Public License 1.0
 
     // For Java 8 Conversions
-    val java8Compat = Def.setting { "org.scala-lang.modules" %% "scala-java8-compat" % java8CompatVersion.value } // Scala License
+    // TODO DOTTY
+    val java8Compat = Def.setting { DottyCompatModuleID("org.scala-lang.modules" %% "scala-java8-compat" % java8CompatVersion.value).withDottyCompat(getVersion()) } // Scala License
```

Comment:
This is a library that probably we want to, eventually, cross-compile as well for Scala 3, cc. @SethTisue

Reply:
yeah, https://github.com/scala/scala-java8-compat/issues/200 ... hoping someone will volunteer to tackle it. if you have any problems using it with `withDottyCompat`, let me know, I would consider that high priority to investigate